src/bpfbytecode.cpp

```diff
@@ -49,7 +49,8 @@ static int btf_type_size(const struct btf_type *t)
     case BTF_KIND_ENUM:
       return base_size + vlen * sizeof(struct btf_enum);
     case BTF_KIND_ENUM64:
-      return base_size + vlen * sizeof(struct btf_enum64);
+      // btf_enum64 is not available until v6.0
```

Comment:
Could we add the UAPI definition of `btf_enum64` to the comment? Just to make it clear why the below `sizeof` expression is used.

Reply:
Sounds good, I will add their definitions.